src/app/models/__tests__/workspace.server.model.spec.ts

```diff
@@ -192,5 +192,37 @@ describe('Workspace Model', () => {
         mongoose.Error.ValidationError,
       )
     })
+
+    describe('getWorkspaces', () => {
+      it('should return empty array when user has no workspaces', async () => {
+        const mockUserId = new ObjectId()
+        const actual = await Workspace.getWorkspaces(mockUserId)
+
+        expect(actual).toEqual([])
+      })
+
+      it('should return array of workspaces belonging to user', async () => {
+        const mockUserId = FORM_ADMIN_USER._id
+        const actual = await Workspace.getWorkspaces(mockUserId)
+
+        expect(actual.length).toEqual(1)
```

Comment:
doesn't seem to be a full test here?

Reply:
Updated the test to expect for match instead!